src/App.js

```diff
@@ -1,24 +1,21 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import SearchMeme from "./Search";
```

Comment:
Put this between header and footer imports? Just for aesthetics as it would be in order.